dotnet/src/SemanticKernel.Abstractions/AI/Embeddings/TextEmbeddingServiceExtensions.cs

```diff
@@ -0,0 +1,135 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.SemanticKernel.AI.Embeddings;
+using Microsoft.SemanticKernel.Services;
+
+// Use base namespace for better discoverability and to avoid conflicts with other extensions.
+namespace Microsoft.SemanticKernel;
```

Comment:
I initially had these extension classes all in their "proper" namespaces, but it made the extension undiscoverable... I had to add additional explicit usings to dozens of files, and intellisense was no help at all.

@dluc I know your IDE will squawk if these files are in these folders with mismatched namespaces. Is there a suppression we can apply? Alternatively, we could put all of these classes in one huge file in the project root, but I think it's kind of nice having them hang out with their relatives.

Reply:
for extensions methods I've been suppressing, there's a few examples in the repo

Reply:
Thanks for the pointers.